src/ctree-element-screen/ctree-segment-container.html

```diff
@@ -187,18 +187,24 @@
 
       previous() {
         if (this._variationIndex <= 0) return;
-        var component = this.shadowRoot.querySelector('#component');
-        component.editing= false;
-        this._variationIndex--;
+        this._componentOff(-1);
       }
 
       next() {
         if (!this.segment || !this.segment.variations || this._variationIndex >= this.segment.variations.length - 1) return;
-        var component = this.shadowRoot.querySelector('#component');
-        component.editing= false;
-        this._variationIndex++;
+        this._componentOff(1);
       }
 
+      _componentOff(incDec){
```

Comment:
I know we talked about this in person, but just to document it here I should have been more clear about what I was referring to.  I was specifically talking about putting `this.shadowRoot.querySelector('#component');` in a helper function so we don't have to keep writing it.  That's only if we can't inject the dynamically created object into `this.$.component`, which would be the best case scenario.

Reply:
Also, pay attention to code format here ;-)